packages/tables/src/examples/overflow-menu.md

```diff
@@ -0,0 +1,98 @@
+```jsx
+const {
+  zdFontSizeBeta,
+  zdFontWeightSemibold,
+  zdSpacingSm
+} = require('@zendeskgarden/css-variables');
+const { Menu, Item } = require('@zendeskgarden/react-menus');
+
+const StyledCaption = styled(Caption)`
+  font-size: ${zdFontSizeBeta};
+  font-weight: ${zdFontWeightSemibold};
+  margin-bottom: ${zdSpacingSm};
+`;
+
+const OverflowMenu = () => (
+  <Menu
+    onChange={selectedKey => alert(selectedKey)}
+    placement="bottom-end"
+    trigger={({ ref, isOpen }) => (
+      <OverflowButton
+        innerRef={ref}
+        active={isOpen}
+        onBlur={e => {
```

Comment:
Focus state should not be seen with mouse-only interaction.

Reply:
This is now fixed in the example.